src/utils/default-rollup-config.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const commonjsPlugin = require('@rollup/plugin-commonjs');
+const jsonPlugin = require('@rollup/plugin-json');
+const replacePlugin = require('@rollup/plugin-replace');
+const resolvePlugin = require('@rollup/plugin-node-resolve');
+
+const plugins = [
+  // @ts-ignore
+  replacePlugin({
+    'process.env.NODE_ENV': `"${process.env.NODE_ENV}"` || '"development"',
+  }),
+  // @ts-ignore
+  resolvePlugin({
+    mainFields: ['browser', 'module', 'main'],
+  }),
+  // @ts-ignore
+  jsonPlugin(),
+  // @ts-ignore
+  commonjsPlugin({
+    sourceMap: false,
+  }),
+];
+
+// Prevent mutation by creating a new object for every require('default-rollup-config.js')
+Object.defineProperty(module, 'exports', {
```

Comment:
Not 100% sure this is kosher, but seems to work!

Reply:
If it breaks one could always change it to `getDefaultRollupConfig` and have that do the copying